.github/pull_request_template.md

```diff
@@ -13,18 +11,13 @@ Describe the tests that you ran to verify your changes. Provide instructions for
 - [ ] Test A (e.g. Test A - New test that ... ran in local, docker, and dev unstable.)
 - [ ] Test B
 
-## Definition of Done
+## PR checklist
 
 Before submitting this PR, please make sure:
 
-- [ ] The work addresses the description and outcomes in the issue
-- [ ] I have added relevant tests for new or updated functionality
-- [ ] My code follows conventions, is well commented, and easy to understand
-- [ ] My code builds and tests pass without any errors or warnings
-- [ ] I have tagged the relevant reviewers
+- [ ] I have tagged the relevant reviewers and interested parties
```

Comment:
can we keep one about "adding or updating relevant tests"?

Reply:
I think that's sufficiently covered by the "How Has This Been Tested?" section